File: CourseProjects/CourseProjects/VisualStudioProjects/Languages/Basic VB Programming/Interfaces/Tester.vb
Module Tester

    Sub Main()
        Dim t As New Triangle
        Dim s As New Square

        testColorable(t)
        testColorable(s)
    End Sub

    Sub testColorable(ByVal ic As IColorable)
        ic.X = 25
        ic.Y = 25
        ic.draw(10)
        ic.paint(255)
    End Sub

End Module

File: CourseProjects/CourseProjects/VisualStudioProjects/Languages/Basic CS  Programming/Shapes/Tester.cs
using System;

namespace Interfaces
{
	class Tester {
		
		static void Main(string[] args) {
			Triangle t = new Triangle();
			Square s = new Square();

			testColorable(t);
			testColorable(s);
		}
		private static void testColorable(IColorable ic) {
			ic.X = 25;
			ic.Y = 25;
			ic.draw(10);
			ic.paint(255);
		}
	}
}
